source/docs/software/software-overview/control-system-software.rst

```diff
@@ -0,0 +1,133 @@
+
+.. image:: images/control-system-software/frc-labview-dashboard.png
+
+The LabVIEW Dashboard is automatically launched by the FRC Driver Station by default. The purpose of the Dashboard is to provide feedback about the operation of the robot using tabbed display with a variety of built in features.  More information about the FRC Default Dashboard software can be found :ref:`here <docs/software/driverstation/driver-station-labview-dashboard:FRC Driver Station LabVIEW Dashboard>`.
+
+SmartDashboard
+^^^^^^^^^^^^^^
+
+.. image:: images/control-system-software/smartdashboard.png
+
+SmartDashboard allows you to view your robot data by automatically creating customizable indicators specifically for each piece of data sent from your robot.  Additional documentation on SmartDashboard can be found :ref:`here <docs/software/wpilib-tools/smartdashboard/index:SmartDashboard>`.
+
+Shuffleboard
+^^^^^^^^^^^^
+
+.. image:: images/control-system-software/shuffleboard.png
+
+Shuffleboard has the same features as SmartDashboard. It also improves on the setup and visualization of your data with new features and a modern design at the cost of being less resource efficient.  Additional documentation on Shuffleboard can be found :ref:`here <docs/software/wpilib-tools/shuffleboard/index:Shuffleboard>`.
+
+FRC roboRIO Imaging Tool (Windows Only)
+---------------------------------------
+
+.. image:: /docs/zero-to-robot/step-3/images/imaging-your-roborio/roborio-imaging-tool.png
+
+This tool is used to format and setup an roboRIO-FRC device for use in FRC. Installation instructions can be found :ref:`here <docs/zero-to-robot/step-2/frc-game-tools:Installing the FRC Game Tools>`. Additional instructions on imaging your roboRIO using this tool can be found :doc:`here </docs/zero-to-robot/step-3/imaging-your-roborio>`.
+
+FRC Radio Configuration Utility (Windows Only)
+----------------------------------------------
+
+.. image:: images/control-system-software/frc-radio-configuration-utility.png
+
+The FRC Radio Configuration Utility is a tool used to configure the standard radio for practice use at home. This tool sets the appropriate network settings to mimic the experience of the FRC playing field. The FRC Radio Configuration Utility is installed by a standalone installer that can be found :ref:`here <docs/zero-to-robot/step-3/radio-programming:Programming your Radio>`.
+
+FRC Driver Station Log Viewer (Windows Only)
+--------------------------------------------
+
+.. image:: images/control-system-software/frc-log-viewer.png
+
+The FRC Driver Station Log Viewer is used to view logs created by the FRC Driver Station. These logs contain a variety of information important for understand what happened during a practice session or FRC match. More information about the FRC Driver Station Log Viewer and understanding the logs can be found :ref:`here <docs/software/driverstation/driver-station-log-viewer:Driver Station Log File Viewer>`
+
+LiveWindow
```

Comment:
Mostly because it had a section before.  I agree that there isn't a ton of merit for it being a completely different software package but I really would like to Drive more people to using the testing features and this might do that.  I can go either way.

Reply:
I could go either way too, just think if it stays it should stick closely to the software it's a part of. It can have it's own section but I think it should be right below the 2 dashboards that use it.

Reply:
I moved the link to right under the other dashboards